packages/metadata-crawler/src/crawler.ts

```diff
@@ -0,0 +1,114 @@
+import { IdentityMetadataWrapper } from '@celo/contractkit'
+import { Client } from 'pg'
+import { ClaimTypes } from '@celo/contractkit/lib/identity'
+import { verifyDomainRecord } from '@celo/contractkit/lib/identity/claims/verify'
+import { normalizeAddress } from '@celo/utils/lib/address'
+
+const PGUSER = process.env['PGUSER'] || 'postgres'
+const PGPASSWORD = process.env['PGPASSWORD'] || ''
+const PGHOST = process.env['PGHOST'] || '127.0.0.1'
+const PGPORT = process.env['PGPORT'] || '5432'
+const PGDATABASE = process.env['PGDATABASE'] || 'blockscout'
+
+const client = new Client({
+  user: PGUSER,
+  password: PGPASSWORD,
+  host: PGHOST,
+  port: Number(PGPORT),
+  database: PGDATABASE,
+})
+
+// read all from table
+// when they were last modified?
+
+async function jsonQuery(query: string) {
+  let res = await client.query(`SELECT json_agg(t) FROM (${query}) t`)
+  // .catch( error => console.error(`Error in jsonQuery: ${error}`))
+  return res.rows[0].json_agg
+}
+
+// async function urlGetter(_str: string) {
+//   return ''
+// }
+
+async function addVerificationClaimToDatabase(address: string, domain: string, verified: boolean) {
+  try {
+    const query = `INSERT INTO celo_claims (address, type, element, verified, timestamp, inserted_at, updated_at) VALUES 
+        (decode($1, 'hex'), 'domain', $2, $3, now(), now(), now()) 
+        ON CONFLICT (address, type, element) DO 
+        UPDATE SET verified=$3, timestamp=now(), updated_at=now() `
+    const values = [address, domain, verified]
+
+    await client
+      .query(query, values)
+      .catch((error) => console.error(`Database error ${error}\n query: ${query}`))
```

Comment:
Can we use `bunyan` similarly to the `attestation-service` to get structured logs by default? https://github.com/celo-org/celo-monorepo/blob/master/packages/attestation-service/src/logger.ts#L1

Reply:
:+1: